manywheel/Dockerfile_2_28

```diff
@@ -0,0 +1,143 @@
+# syntax = docker/dockerfile:experimental
+ARG ROCM_VERSION=3.7
+ARG BASE_CUDA_VERSION=10.2
+ARG GPU_IMAGE=nvidia/cuda:${BASE_CUDA_VERSION}-devel-centos7
```

Comment:
Please note: GPU_IMAGE building here is placeholder, we are not building it now.